t/007-clear-cache.t

```diff
@@ -0,0 +1,73 @@
+#!perl
+
+use strict;
+use warnings FATAL => 'all';
+
+use Test::More tests => 14;
+
+use constant {
+    ADDRESS => 'wikipedia.org',
+    DIFFERENT_ADDRESS => 'en.wikipedia.org',
+    SCHEME => 'https://',
```

Comment:
Since the `SCHEME` never changes, we can just prefix the two addresses with it and have one less constant.

Reply:
But we need the addresses without the scheme for Test::RequiresInternet...

Reply:
In that case, please ignore my suggestion. :)